shotover-proxy/src/transforms/tee.rs

```diff
@@ -167,3 +186,83 @@ impl Transform for Tee {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::config::topology::TopicHolder;
+    use crate::transforms::TransformsConfig;
+    use std::collections::HashMap;
+
+    #[tokio::test]
+    async fn test_validate_no_subchain() {
+        let holder = TopicHolder {
+            topics_rx: HashMap::new(),
+            topics_tx: HashMap::new(),
+        };
+
+        let mut config = TeeConfig {
```

Comment:
Can we wrap each of these in their own block instead of making the variables mutable? that way its easier to reason about.
e.g.
```rust
{
       let config = TeeConfig {
            behavior: Some(ConsistencyBehavior::Ignore),
            timeout_micros: None,
            chain: vec![TransformsConfig::Null],
            buffer_size: None,
        };
        let transform = config.get_source(&holder).await.unwrap();
        let result = transform.validate();
        assert_eq!(result, Vec::<String>::new());
}

```